### PR TITLE
Snowflake sharing update

### PR DESCRIPTION
**Pull request by kbl-tomas-hegedus**
Jira issue(s): PROOF-XXX

Changes:

-

---

### CODE REVIEW

**Review by hhanova — APPROVED** [review]

File: storage/byodb/external-buckets/index.md [hunk 1]
```diff
@@ -6,40 +6,37 @@ permalink: /storage/byodb/external-buckets/
...
 
 ### Snowflake
 Fill in the **name** of the new bucket, **database** name, and **schema** name. Click **Next Step**. Keboola will then generate a code that you can use to 
 grant Keboola correct access to the schema in your Snowflake. Once access has been granted, click **Register Dataset** to start using it.
 
 {: .alert.alert-info}
-Note: This set of permissions grants the Keboola service account read-only access to the data.
-
-{: .alert.alert-warning}
-[Sharing data via Snowflake](https://docs.snowflake.com/en/guides-overview-sharing) is currently not supported. Attempting to grant permissions will result in an error.
```
Comment (jirkasemmler) [inline storage/byodb/external-buckets/index.md:35]: @kbl-tomas-hegedus toto porad plati . toto se bude resit az v https://keboola.atlassian.net/browse/CT-1393

File: storage/byodb/external-buckets/index.md [hunk 2]
```diff
@@ -91,9 +89,15 @@ WHERE `source` = "mql";
...
 {: .alert.alert-warning}
-**Note:** The dataset name is the name of the bucket you provided in the dialog (`users-reporting`), not that of the original dataset created in BigQuery 
+**Note:** The dataset name is the name of the bucket you provided in the dialog (`users-reporting`), not the original dataset created in BigQuery 
 (`sales_emea`). However, there are no technical limitations; they can have the same name. 
 
+## Sharing an External Dataset
+It is possible to share a Snowflake external bucket using the same process as [any other Storage bucket](https://help.keboola.com/catalog/#enable-sharing). Once the bucket is shared, the refresh operation is only available in the source project. Currently, it is possible to share entire buckets, not specific tables within them.
+
+{: .alert.alert-warning}
+**Note:** At this moment, it is not possible to share BigQuery or BigLake datasets. If this is a relevant use case for you, please create a support ticket.
```
Comment (jirkasemmler) [inline storage/byodb/external-buckets/index.md:99]: biglake bych do toho vubec netahal.

File: storage/byodb/external-buckets/index.md [hunk 3]
```diff
@@ -91,9 +89,15 @@ WHERE `source` = "mql";
 ```
 
 {: .alert.alert-warning}
-**Note:** The dataset name is the name of the bucket you provided in the dialog (`users-reporting`), not that of the original dataset created in BigQuery 
+**Note:** The dataset name is the name of the bucket you provided in the dialog (`users-reporting`), not the original dataset created in BigQuery 
 (`sales_emea`). However, there are no technical limitations; they can have the same name. 
 
+## Sharing an External Dataset
+It is possible to share a Snowflake external bucket using the same process as [any other Storage bucket](https://help.keboola.com/catalog/#enable-sharing). Once the bucket is shared, the refresh operation is only available in the source project. Currently, it is possible to share entire buckets, not specific tables within them.
```
Comment (jirkasemmler) [inline storage/byodb/external-buckets/index.md:96]: `...source project (the project where the external dataset has been registered in)`

Comment (jirkasemmler) [reply storage/byodb/external-buckets/index.md:96]: bych trochu v te zavorce dovysvetlil co znamena "source project"

**Review by jirkasemmler — COMMENTED** [review]
je tam par nejasnosti, ktere doporucuju upravit.

**Pull request merged**